autogpt/config/config.py

```diff
@@ -55,8 +55,20 @@ def __init__(self) -> None:
         )
         self.fast_llm_model = os.getenv("FAST_LLM_MODEL", "gpt-3.5-turbo")
         self.smart_llm_model = os.getenv("SMART_LLM_MODEL", "gpt-4")
-        self.fast_token_limit = int(os.getenv("FAST_TOKEN_LIMIT", 4000))
-        self.smart_token_limit = int(os.getenv("SMART_TOKEN_LIMIT", 8000))
+
+        """ Load here to avoid circular imports. """
+        from autogpt.llm.providers.openai import OPEN_AI_CHAT_MODELS
+
+        self.OPEN_AI_CHAT_MODELS = OPEN_AI_CHAT_MODELS
```

Comment:
Why is this needed?

Reply:
since I decided to DRY things up and use a function, I needed a reference to the chat models. I could have passed it into the `get_max_token_limit` function but decided to assign it to the Config class instead